### PR TITLE
DOC: Add Chinese doc for external storage JuiceFS on K8S

### PR DESCRIPTION
**Pull request by Zhou1213CN**
## Related issue number

Fixes #xxxx

## Check code requirements

- [ ] tests added / passed (if needed)
- [ ] Ensure all linting tests pass

### CODE REVIEW

File: doc/source/locale/zh_CN/LC_MESSAGES/user_guide/deployment_kubernetes.po [hunk 1]
```diff
@@ -217,3 +217,398 @@ msgstr ""
 "<https://docs.conda.io/projects/conda/en/latest/user-"
 "guide/concepts/channels.html>`_。"
 
+#: ../../source/user_guide/deployment_kubernetes.rst:104
+msgid ""
+"Xorbits is now able to integrate `JuiceFS <https://juicefs.com/en/>`_, a distributed POSIX file system that can be easily integrated with Kubernetes to provide persistent storage, as one of the storage backend."
+msgstr ""
+"Xorbits现在集成了 `JuiceFS <https://juicefs.com/zh-cn/>`_  ，一个可以与Kubernetes轻松集成来提供持久化存储的分布式POSIX文件系统。"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/user_guide/deployment_kubernetes.po:224]: You may want to add a spece between a chinese char and a english char or a number.
```suggestion
"Xorbits 现在集成了 `JuiceFS <https://juicefs.com/zh-cn/>`_  ，一个可以与 Kubernetes 轻松集成来提供持久化存储的分布式 POSIX 文件系统。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/user_guide/deployment_kubernetes.po [hunk 2]
```diff
@@ -217,3 +217,398 @@ msgstr ""
...
+msgid ""
+"You can also check how much CPU and memory resources redis pod gets by running"
+msgstr ""
+"您可以通过运行以下命令来查看Redis Pod获得了多少CPU和内存资源："
+
+#: ../../source/user_guide/deployment_kubernetes.rst:152
+msgid ""
+"and check the corresponding fields."
+msgstr ""
+"检查相应的字段。"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/user_guide/deployment_kubernetes.po:284]:
```suggestion
"并检查相应的字段。"
```

File: doc/source/user_guide/deployment_kubernetes.rst [hunk 3]
```diff
@@ -418,7 +451,7 @@ Deploy Xorbits cluster, for example:
     from xorbits.deploy.kubernetes
     import new_cluster
```
Comment (UranusSeven) [inline doc/source/user_guide/deployment_kubernetes.rst:452]: `from xorbits.deploy.kubernetes import new_cluster` looks better.

**Review by UranusSeven — APPROVED** [review]

**Review by aresnow1 — APPROVED** [review]
LGTM

**Pull request merged**